packages/docs/src/components/Key.jsx

```
@@ -0,0 +1,69 @@
+
+  let keys = [...mods, title(k)];
+
+  return (
+    <div style={{ display: 'inline-flex' }}>
+      {keys.map(
+        (key, idx) => (
+          <div
+            className={classes.key}
+            style={{ marginLeft: idx === 0 ? 0 : '0.25rem' }}
+          >
+            {key}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)